crates/relayer-types/src/applications/transfer/mod.rs

```diff
@@ -23,4 +23,4 @@ pub const MODULE_ID_STR: &str = "transfer";
 pub const PORT_ID_STR: &str = "transfer";
 
 /// ICS20 application current version.
-pub const VERSION: &str = "ics20-1";
+pub const VERSION: &str = "ics20";
```

Comment:
```suggestion
pub const VERSION_PREFIX: &str = "ics20";
```

Reply:
done in fc41cdceb1b3993243cbf406faf415bc098f2599